page/code-organization/deferreds/jquery-deferreds.md

```diff
@@ -37,9 +37,9 @@ $.when(
 ).then( successFunc, failureFunc );
 ```
 
-The `$.when()` implementation offered in jQuery is quite interesting as it not only interprets deferred objects, but when passed arguments that are not deferreds, it treats these as if they were resolved deferreds and executes any callbacks (doneCallbacks) right away. It is also worth noting that jQuery's deferred implementation, in addition to exposing deferred.then(), a jQuery promise also supports the deferred.done() and deferred.fail() methods which can also be used to add callbacks to the deferred's queues.
+The `$.when()` implementation offered in jQuery is quite interesting as it not only interprets deferred objects, but when passed arguments that are not deferreds, it treats these as if they were resolved deferreds and executes any callbacks (doneCallbacks) right away. It is also worth noting that jQuery's deferred implementation, in addition to exposing deferred.then(), also supports the deferred.done() and deferred.fail() methods which can also be used to add callbacks to the deferred's queues.
```

Comment:
Minor point: doneCallbacks, deferred.then(),  deferred.done(), and deferred.fail() should be wrapped by the backtik.